Controller/SiteAdminController.php

```diff
@@ -30,13 +31,13 @@ class SiteAdminController extends Controller
      * @throws NotFoundHttpException
      * @throws AccessDeniedException
      */
-    public function snapshotsAction()
+    public function snapshotsAction(Request $request)
```

Comment:
This is a BC-break for extending classes

Reply:
I think `CRUDController` of admin-bundle handles the request with a different maner.

Reply:
https://github.com/sonata-project/SonataAdminBundle/blob/c39403f1b801cf2d40d916d2cd00daf753adeec9/Controller/CRUDController.php#L948-L955